site/en/tutorials/images/transfer_learning.ipynb

```diff
@@ -69,7 +69,7 @@
         "id": "hRTa3Ee15WsJ"
```

Comment:
<p> </p><p>It's always better to show the number than write it in the text, so nobody makes a bug if it comes out to be 97 or 99%</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/2081/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='86'/>

Reply:
<p>The <code>Model.fit</code> output after 20 epochs and TensorBoard's UI should be enough then. So:</p><p> </p><p>"After fine-tuning, the model should reach a high accuracy on the validation set."</p>